### PR TITLE
OEBS map

### PR DESCRIPTION
**Pull request by Eulade**
In this PR: 

- New map with associated xsl file - OEBS e2b to e2bInterchangei. 
- Bindings with a receive port for OEBS invoices

### CODE REVIEW

File: Invoice.OEBS/Invoice.OEBS.Transforms/xslt/mapE2bV3p41OebsToE2bInterchangei.xsl [hunk 1]
```diff
@@ -0,0 +1,468 @@
...
+				</xsl:element>
+			</xsl:if>
+		</xsl:element>
+	</xsl:template>
+
+	<!-- References -->
+	<xsl:template match="s0:REFERENCE">
+		<xsl:element name="Reference">
+			<xsl:element name="ReferenceNo">
+				<xsl:value-of select="s0:FREE_REFERENCE_CODE" />
```
Comment (MariusLindberg) [inline Invoice.OEBS/Invoice.OEBS.Transforms/xslt/mapE2bV3p41OebsToE2bInterchangei.xsl]: Is this correct? In the code FREE_REFERENCE_CODE is mapped to ReferenceNo and FREE_REFERENCE_CONTENT is mapped to referenceType?

File: Invoice.OEBS/Invoice.OEBS.Transforms/xslt/mapE2bV3p41OebsToE2bInterchangei.xsl [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+							</xsl:attribute>
+							<xsl:attribute name="idType">
+								<xsl:text>EANSSCC</xsl:text>
+							</xsl:attribute>
+						</xsl:if>
+						<xsl:if test="s0:WEIGHT !=''">
+							<xsl:element name="TotalGrossWeight">
+								<xsl:attribute name="unitCode">
+									<xsl:choose>
+										<xsl:when test="not(s0:WEIGHT_UOM) or s0:WEIGHT_UOM =''">
```
Comment (MariusLindberg) [inline Invoice.OEBS/Invoice.OEBS.Transforms/xslt/mapE2bV3p41OebsToE2bInterchangei.xsl]: Just a comment. The test is redundant. s0:WEIGHT_UOM  != '' would have checked for both existence of node and content in field. Does not need to be changed. Just a comment. Readability is OK. 

Also possible to check for existence and value in field by doing. I THINK this should not hit on missing and blank nodes.
if(s0:WEIGHT_UOM/text())

File: Invoice.OEBS/Invoice.OEBS.Transforms/xslt/mapE2bV3p41OebsToE2bInterchangei.xsl (hunk 1 above)
Comment (Eulade) [inline Invoice.OEBS/Invoice.OEBS.Transforms/xslt/mapE2bV3p41OebsToE2bInterchangei.xsl]: Thank you, this was just mixed up. Changed

File: Invoice.OEBS/Invoice.OEBS.Transforms/xslt/mapE2bV3p41OebsToE2bInterchangei.xsl (hunk 2 above)
Comment (Eulade) [inline Invoice.OEBS/Invoice.OEBS.Transforms/xslt/mapE2bV3p41OebsToE2bInterchangei.xsl]: Noted. Will clean up later...

**Comment by MariusLindberg** [comment]
I see there are other people than me working late :smile: 

The only thing I was unsure of was a bug is fixed. Ignoring my comment and merging the branch since a fix has been pushed.

**Pull request merged**